cel/cos_tlv.go

```diff
@@ -14,6 +14,18 @@ const (
 	CosEventType uint8 = 80
 	// CosEventPCR is the PCR which should be used for CosEventType events.
 	CosEventPCR = 13
+	// CosRTMR is the RTMR to be extended for COS events
+	// According to https://uefi.org/specs/UEFI/2.10/38_Confidential_Computing.html
+	// Eventlog Index      TDX Register
+	// 0                   MRTD
+	// 1                   RTMR[0]
+	// 2                   RTMR[1]
+	// 3                   RTMR[2]
+	// So:
+	// 4                   RTMR[3]
+	CosRTMR = 3
+	// CosEventLogRegisterIndex is the register index to use in eventlog.
+	CosEventLogRegisterIndex = 4
```

Comment:
nit: CosCCELMRIndex. The existing comment is still accurate, but you could explain a bit more the CCELMRIndex

Reply:
done